website/data/users.yml

```diff
@@ -177,3 +177,6 @@
 - caption: 3YOURMIND
   image: "/images/users/3yourmind.svg"
   infoLink: https://www.3yourmind.com/
+- caption: Leesa Sleep
+  image: "/images/users/leesa.svg"
```

Comment:
no need for the `"`

Reply:
maybe you can also remove them from line 178 as well..

Reply:
Meh, let's just merge and deal with quotes some other time.